frontend/src/views/MLStudio/MLStudioEditForm.js

```diff
@@ -89,7 +89,7 @@ const PipelineEditForm = (props) => {
           variant: 'success'
         });
```

Comment:
I can find a few places where `sagemakerStudioUserProfileUri` is used. Is it intended or should we rename them as well?